scripts/check

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+
+export PREFIX=""
+if [ -d 'venv' ] ; then
+    export PREFIX="venv/bin/"
+fi
+export SOURCE_FILES="typesystem tests"
+
+set -x
+
+export SOURCE_FILES="typesystem tests"
+
+${PREFIX}isort --check --diff --project=typesystem $SOURCE_FILES
+${PREFIX}black --check --diff $SOURCE_FILES
+${PREFIX}flake8 $SOURCE_FILES
+# ${PREFIX}mypy $SOURCE_FILES
```

Comment:
You can _uncomment_ and add the same `files` as `uvicorn`: https://github.com/encode/uvicorn/blob/ad761362b49278606520a1ae4712f68c8016cfb5/setup.cfg#L5-L44

If you want.

Reply:
@Kludex thanks for that. Is there anything else you think should be changed?